pkg/api/templates/swagger.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Swagger UI</title>
```

Comment:
I don't understand why we bring the Swagger UI front page here, as this should be served statically by swagger already. Can we avoid it somehow? why is that?

Reply:
I don't know -- If you have a different better way, lmk.